src/system/Form/RadioBoxGroup.js

```diff
@@ -94,16 +94,102 @@ const RadioOption = ( {
 	);
 };
 
-RadioOption.propTypes = {
+const ChipOption = ( {
+	defaultValue,
+	option: { id, value, label },
+	name,
+	disabled,
+	onChangeHandler,
+} ) => {
+	const checked = `${ defaultValue }` === `${ value }`;
+	const forLabel = id || value;
+	const ref = React.useRef( null );
+	const describedById = `input-radio-box-${ forLabel }-description`;
+
+	return (
+		<div
+			id={ `o${ forLabel }` }
+			onClick={ () => {
+				ref.current?.click();
+			} }
+			sx={ {
+				display: 'inline-flex',
+				position: 'relative',
+				background: checked ? 'layer.4' : undefined,
+				color: 'text',
+				minHeight: '32px',
+				boxShadow: checked ? 'low' : undefined,
+				'&:hover': {
+					background: checked ? 'layer.4' : 'layer.1',
+				},
+				borderRadius: 1,
+			} }
+		>
+			<input
+				ref={ ref }
+				type="radio"
```

Comment:
```suggestion
				type="radio"
				id={ forLabel }
```
Currently we have `htmlFor={ forLabel }` on the label but it doesn't correspond to an input id. This should fix that.